rules.js

```diff
@@ -100,7 +101,10 @@ module.exports = function(styles) {
         state.withinText = true;
         return React.createElement(Text, {
           key: state.key,
-          style: styles.autolink
+          style: styles.autolink,
+          onPress: function () {
+            Linking.openURL(node.content[0].content);
```

Comment:
(it also may or may not be worth catching the error callback for `Linking.openURL`; I haven't used that to know how often it happens.)

It *might* also be useful to check for `//` links and change them to `https://` links, as I'm not sure `Linking.openURL` can handle them.